application/controllers/items.py

```diff
@@ -79,9 +80,8 @@
 item_post_parser.add_argument(
     'status',
     type=str,
-    required=True,
-    location=['json'],
-    help='status parameter is required'
+    required=False,
+    location=['json']
 )
```

Comment:
Good update. Because this field is never editable by humans & will only be updated by methods in the app, no validation neccessary.